applications/structural_application/custom_python/structural_python_application.cpp

```diff
@@ -75,72 +76,71 @@ namespace Kratos
 namespace Python
 {
 
-using namespace boost::python;
-
+using namespace pybind11;
```

Comment:
```suggestion
namespace py = pybind11
```

Otherwise there are problems in Windows due to redefinitions of some things